eth/ethconfig/config.go

```diff
@@ -167,10 +167,8 @@ func CreateConsensusEngine(config *params.ChainConfig, db ethdb.Database) (conse
 	if config.Taiko {
```

Comment:
If `--taiko` is set, the error below won't be returned?

Reply:
Yeah, if the `taiko` flag is used geth will [TaikoChainConfig](https://github.com/taikoxyz/taiko-geth/blob/7bf5c0d259f60f9d62d481c873053548c87b6fb5/params/taiko_config.go#L79) and then the check is true.